src/AstGen.zig

```diff
@@ -4644,7 +4645,7 @@ fn testDecl(
             break :blk name_str_index;
         }
         // String table index 1 has a special meaning here of test decl with no name.
-        break :blk 1;
+        break :blk @enumFromInt(1);
```

Comment:
```suggestion
        break :blk .unnamed_test_decl;
```

and you can delete the comment above now too!